vscode/src/client.ts

```diff
@@ -360,7 +360,7 @@ export default class Client extends LanguageClient implements ClientInterface {
             name: error.data.errorClass,
             stack,
           },
-          { ...error.data },
+          { ...error.data, serverVersion: this.serverVersion },
```

Comment:
I can't remember if we report telemetry events while working on ruby-lsp itself? If so, should we have some way to distinguish these?